spec/features/bucket_lists/user_edits_bucket_lists_spec.rb

```diff
@@ -12,14 +12,35 @@
 
 
   scenario "user successfully changes the title of a bucket list" do
+    bucket_list = FactoryGirl.create(:bucket_list)
+
     visit new_user_session_path
-    sign_in
-    visit root_path
-    click_link 'New Bucket List!'
-    fill_in 'Title', with: 'Europe'
-    click_button 'Create Your Bucket List!'
-
-    expect(page).to have_content("Congrats! You started a new Bucket List!")
-    expect(page).to have_content('Europe')
+    fill_in 'Email', with: bucket_list.user.email
+    fill_in 'Password', with: bucket_list.user.password
+    click_button 'Log in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.